LINUX/netmap_linux.c

```diff
@@ -1105,6 +1105,12 @@ nm_os_generic_find_num_desc(struct ifnet *ifp, unsigned int *tx, unsigned int *r
 			*tx = netmap_generic_ringsize;
 		}
 		error = 0;
+	}else{
```

Comment:
This is a generic function, we cannot put this stuff here. If you need these values please create your own ```nm_config``` callback.

Reply:
need to check if  following function below can actually set the values correctly
static int stmmac_netmap_config(struct netmap_adapter *na,
				struct nm_config_info *info)

Reply:
ack